CountryPicker/CountryPicker/PickerView/CountryPickerView.swift

```diff
@@ -129,7 +129,8 @@ extension CountryPickerView: UIPickerViewDelegate {
             reuseableView = ComponentView(frame: rect)
         }
         
-        let country = CountryManager.shared.countries[row]
+//         let country = CountryManager.shared.countries[row]
+        let country = pickList[row]
```

Comment:
why you had left commented code? 
Any reason for the change, please

Reply:
No reason at all. Do you want me to remove the commented lines?

Reply:
(Done)